libcxx/docs/ReleaseNotes/22.rst

```diff
@@ -45,6 +45,8 @@ Improvements and New Features
 
 - The performance of ``map::map(const map&)`` has been improved up to 2.3x
 - The performance of ``map::operator=(const map&)`` has been improved by up to 11x
+- The performance of ``map::erase`` and ``set::erase`` has been improved by up to 2x
+- The performance of ``find(key)`` in all associative containers has been improved by up to 2.3x
```

Comment:
```suggestion
- The performance of ``find(key)`` in ``map``, ``set``, ``multimap`` and ``multiset`` has been improved by up to 2.3x
```

Otherwise it's unclear whether `flat_map` and `unordered_map` & friends are also counted